crates/primitives/genesis-config/Cargo.toml

```diff
@@ -17,7 +17,7 @@ blockifier = { workspace = true, features = [
 ] }
 #
 # third party
-derive_more = { workspace = true }
+derive_more = { workspace = true, features = ["constructor"] }
```

Comment:
what's that used for ?

Reply:
I got an error when compiling the crate. 

It was used here:
```
#[derive(Constructor)]
pub struct GenesisLoader {
    base_path: PathBuf,
    data: GenesisData,
}
```

But the feature was not present